### DISCUSSION TITLE
Monitoring request sent bytes

### DISCUSSION THREAD

**Discussion by michelsciortino** [encode/httpcore#489] [Q&A]
I'd like to monitor the sending of chunks of the request body but reading the source code I can't figure how to do this.

The scenario is a stream() operation where a big file is sent through a POST and a progress bar has to display the status of the upload.

**Comment by lovelydinosaur** [comment]
Hiya @michelsciortino.

Can probably help you work through that. Want to share a simplified version of how you're currently sending the upload, and we can work through what a good solution would be for ya?
